vars.md

```diff
@@ -111,6 +111,8 @@ assured that they are persisted.
 | BUILD_CAPI_LOCALLY | build Cluster API based on CAPIPATH | "true","false" | "false" |
 | BUILD_IRONIC_IMAGE_LOCALLY | build the Metal3's ironic-image based on IRONIC_IMAGE_PATH | "true","false" | "false" |
 | IRONIC_FROM_SOURCE | installs ironic from source during container image building, if `true` then the `BUILD_IRONIC_IMAGE_LOCALLY` will be also set to `true` | "true","false" | "false" |
+| DHCP_HOSTS | A list of `;` separated dhcp-host directives for dnsmasq | e.g. `00:20:e0:3b:13:af;00:20:e0:3b:14:af` | |
+| DHCP_IGNORE | A set of tags on hosts to be ignored by dnsmasq | e.g. `tag:!known` | |
```

Comment:
are these related to this series of kustomization PRs?

Reply:
No this is for running multiple instances in the same network. Useful for example in a baremetal lab where multiple hosts are on the same network. Then we can simply list the MACs that we care about and tell dnsmasq to ignore the rest.

Reply:
It is related this PR that I also linked in the description: https://github.com/metal3-io/ironic-image/pull/394

Reply:
ah sorry, I confused this PR with https://github.com/metal3-io/metal3-dev-env/pull/1086